### ISSUE TITLE
Narrator/NVDA not announcing the status message that number of search results that are appearing for search text field.: A11y_Awesome AZD_ Template library_Search_Status message.

### ISSUE THREAD

**Issue by Harika1702** [Azure/awesome-azd#344]
### Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #WCP; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #Feature_AwesomeAZD_Web_Mar2024; #Status Message; #DesktopWeb; #Screen Reader; #NVDA; #Narrator; #STP; #Chrome; #A11yWCAG2.1; #WCAG4.1.3;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge: Version 123.0.2420.20 
Chrome Version 122.0.6261.131 (Official Build) (64-bit)
Narrator/NVDA Version 2023.3.3
Windows 11 Enterprise Version 23H2 OS build 22631.2050.
### Repro Steps: 
1. Turn on narrator/NVDA.
2. Hit this URL: https://azure.github.io/awesome-azd/.
3. Tab till "Template library' section.
4. Tab till "Search" field.
5. Now type anything in the “Search field”.
6. Now verify the issue that Narrator/NVDA not announcing the status message that number of search results that are appearing for search text field or not.
### Actual Result:
Narrator/NVDA not announcing the status message that number of search results that are appearing for search text field.
### Expected Results:
Narrator/NVDA should announce the status message that number of search results that are appearing for search text field.
### User Impact:
Screen reader users will not be able to get the search results information if the status message is not being announced by screen reader.

### Attachment:
<img width="960" alt="344_A11y_Code snippet" src="https://github.com/Azure/awesome-azd/assets/100270074/15061241-bfc1-46cc-b1a8-a3c5defe8514">

https://github.com/Azure/awesome-azd/assets/100270074/1ec62d9c-5b28-49a6-ae03-91b6b4625c55

https://github.com/Azure/awesome-azd/assets/100270074/443b6935-c0d2-4039-b39e-161e2b6f6780

**Comment by hemarina** [comment]
Closing this issue as we made a whole update on the site, and this issue is not update to date with existing site. @Harika1702 Feel free to reopen this issue if needed.
